src/scss/06_components/paragraphs/_event-list.scss

```diff
@@ -0,0 +1,181 @@
+.component--event-list .event {
+  border-bottom: 1px solid #e5e5e5;
+  padding-bottom: $spacing-half;
+  padding-top: $spacing-and-half;
+
+  @include breakpoint($breakpoint-l) {
+    padding-bottom: $spacing;
+    padding-top: $spacing;
+  }
+
+  &:first-child {
+    border-top: 1px solid #e5e5e5;
+  }
+
+  > a {
+    display: block;
+    text-decoration: none;
+
+    &:hover {
+      h3 {
+        text-decoration: underline;
+      }
+    }
+
+    @include breakpoint($breakpoint-l) {
+      display: grid;
+      grid-template-columns: 325px 1fr;
+    }
+  }
+
+  h3 {
+    margin-bottom: $spacing-and-half;
+    margin-top: $spacing;
+  }
+
+  .event__image-container {
+    background-position: center center;
+    background-size: cover;
+    display: block;
+    height: 253px;
+    position: relative;
+    width: 100%;
+
+    @include breakpoint($breakpoint-l) {
+      height: auto;
+    }
+  }
+
+  .event__image {
+    height: 100%;
+    width: 100%;
+
+    @include breakpoint($breakpoint-l) {
+      position: absolute;
+    }
+  }
+
+  img.event__image {
+    object-fit: cover;
+  }
+
+  .event__image.image-placeholder {
+    padding: 0;
+  }
+
+  .event__content-container {
+    @include breakpoint($breakpoint-l) {
+      padding-left: $spacing-double;
+    }
+  }
+
+  .event__tags {
+    gap: $spacing;
+    width: 100%;
+  }
+
+  .event__tags--mobile {
+    bottom: 0;
+    display: flex;
+    flex-wrap: wrap-reverse;
+    justify-content: flex-end;
+    padding: 0 $spacing $spacing;
+    position: absolute;
+
+    @include breakpoint($breakpoint-l) {
+      display: none;
+    }
+  }
+
+  .event__tags--desktop {
+    display: none;
+
+    @include breakpoint($breakpoint-l) {
+      align-items: flex-start;
+      display: flex;
+      flex-wrap: wrap;
+    }
+  }
+
+  .event__tags span {
+    background: #dedfe1;
+    border-radius: 24px;
+    display: inline-block;
+    font-size: var(--fontsize-body-s);
+    font-weight: $font-weight-medium;
+    padding: 5px 12.5px;
```

Comment:
These padding values could be changed to something more generic.

Also font-sizes and font-weights are not supposed to be added to css. Ready made font styles should be added here i.ex. icon_label looks like it could work here.